include/BSMPT/models/SMparam.h

```diff
@@ -227,6 +227,16 @@ const std::complex<double> II(0, 1);
  */
 const ISMConstants GetSMConstants();
 
+/**
+ * @brief speed of sound
+ */
+const double Csound = 0.5773502691896258; // 1/sqrt(3)
+
+/**
+ * @brief reduced Planck mass = MPl / (8 Pi)
+ */
+const double MPl = 2.4e18;
```

Comment:
This should be part of the `ISMConstants` like the other SM constants

Reply:
Done.